src/utils/runtime/src/logger.rs

```diff
@@ -202,8 +202,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
         // Configure levels for external crates.
         filter = filter
             .with_target("foyer", Level::WARN)
-            .with_target("aws_sdk_ec2", Level::INFO)
-            .with_target("aws_sdk_s3", Level::INFO)
+            .with_target("aws", Level::INFO)
```

Comment:
`aws_smithy_http` is too verbose under debug mode (with default level set to `DEBUG`), especially that it yields spans with `DEBUG` level frequently.